devops/scripts/benchmarks/utils/utils.py

```diff
@@ -202,6 +202,37 @@ def get_device_architecture(additional_env_vars):
     return architectures.pop()
 
 
+def warn_if_level_zero_is_not_found(additional_env_vars) -> bool:
+    warning_found = False
+    sycl_ls_found_l0 = False
+
+    sycl_ls_output = run(
+        ["sycl-ls"], add_sycl=True, env_vars=additional_env_vars
+    ).stdout.decode()
+
+    for line in sycl_ls_output.splitlines():
+        if "level_zeroo" in line:
```

Comment:
double `o`? same below

Reply:
Haha, good catch, I was testing. At least now we know warnings are working: https://github.com/intel/llvm/actions/runs/20598926970/job/59159640765?pr=20980#step:19:1093 (trying to find "level_zeroo" which will not succeed, of course).

Fixed.